verify.go

```diff
@@ -205,6 +205,10 @@ func parseClaim(raw []byte, name string, v interface{}) error {
 //
 //    token, err := verifier.Verify(ctx, rawIDToken)
 //
+// Note that AWS Cognito access tokens include client_id instead of aud, so to verify a
```

Comment:
Mind dropping this? We don't call out other providers like this. I can follow up with a doc change for the Config struct to specify that it's checking the "aud" claim for the client ID.

Reply:
Is there a reason this wasn't dropped?